cli/tests/pcluster/validators/test_all_validators/test_scheduler_plugin_all_validators_are_called/scheduler_plugin_1.yaml

```diff
@@ -235,6 +235,7 @@ SharedStorage:
       StorageType: HDD  # HDD | SSD
 Iam:
   PermissionsBoundary: arn:aws:iam::aws:policy/boundary
+  ResourcePrefix: /boundary/role-prefix
```

Comment:
**General best practice**
I suggest to replace `/boundary/role-prefix` with something like `/path/role-prefix` or `/whatever-path/whatever-role-prefix` because we should not suggest the idea that the prefix used in `ResourcePrefix` should match the prefix used by the `PermissionBoundary`.

Let's apply this best practices in every other test parameter in this PR.

Reply:
I will change it to /path-prefix/role-prefix

Reply:
Not clear to me: when we put `ResourcePrefix` is `/blablabla/tiptoe`, then `blablabla` will be the path or the path prefix?

Reply:
Will rename to /path-prefix/name-prefix